info/lib/display.js

```diff
@@ -1,5 +1,7 @@
 "use strict";
 
+const yaml = require('yaml');
```

Comment:
Can we use [`js-yaml`](https://www.npmjs.com/package/js-yaml) for this? We already have it in the list of dependencies.